spec/features/clearance/visitor_signs_up_spec.rb

```diff
@@ -1,31 +1,43 @@
 # frozen_string_literal: true
 
-require "rails_helper"
-require "support/features/clearance_helpers"
+require 'rails_helper'
+require 'support/features/clearance_helpers'
 
-RSpec.feature "Visitor signs up" do
-  scenario "by navigating to the page" do
+RSpec.feature 'Visitor signs up' do
+  scenario 'by navigating to the page' do
     visit sign_in_path
 
-    click_link I18n.t("sessions.form.sign_up")
+    click_link I18n.t('sessions.form.sign_up')
 
     expect(current_path).to eq sign_up_path
   end
 
-  scenario "with valid email and password" do
-    sign_up_with "testname", "valid@example.com", "password"
+  scenario 'with valid email and password' do
+    sign_up_with 'testname', 'valid@example.com', 'password'
```

Comment:
Does the linter/style-tool youre using require single quotes? If so, I think thats perfectly fine in your own app, but I would like to point out the [thoughtbot style guide](https://github.com/thoughtbot/guides/tree/master/style/ruby) on this:

> Prefer double quotes for strings.

By no means is this a request for change :). But for client apps, we try to follow the above styles as much as we can.

Reply:
Thanks for pointing that out! I am actually getting conflicting messages from Hound and Rubocop. Hound says 'single-quote' and Rubocop says 'double-quote' I'll fix these to double quote.